api/src/controllers/users.js

```diff
@@ -233,7 +233,12 @@ module.exports = {
   v2: {
     get: async (req, res) => {
       try {
-        const body = await getUserList(req, res);
+        await auth.check(req, 'can_view_users');
+        const filters = _.chain(req.query)
+          .pick(['facility_id', 'contact_id']) // keep only supported filtering properties
+          .mapKeys((value, key) => _.camelCase(key)) // rename them to keep consistent variable names
+          .value();
```

Comment:
This filters out any property we don't support filtering users on and renames them to camelCase to keep consistent variable names

```ts
{ whatever: true, contact_id: '123' } // => { contactId: '123' }
```